CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.6.2
+ - Fix bug where update index actions would not work with events with 'data' field
```

Comment:
or, sorry. looks like this fix does the opposite. removes this from the behavior. ignore above